src/ngraph/runtime/gpu/CMakeLists.txt

```diff
@@ -72,8 +72,32 @@ if (NGRAPH_GPU_ENABLE)
 	    -gencode=arch=compute_70,code=compute_70)
     endif()
 
-    set_source_files_properties( ${CUDA_SRC} PROPERTIES CUDA_SOURCE_PROPERTY_FORMAT OBJ)
-    cuda_compile(CUDA_OBJ ${CUDA_SRC} STATIC)
+    set (DO_CUDA_COMPILE FALSE)
+    if (CUDA9_FOUND)
+        set (DO_CUDA_COMPILE TRUE)
```

Comment:
no need this one?

Reply:
Not just no need, it is incorrect. Remove line 77

Reply:
Will fix.